bin/cli.js

```diff
@@ -2,12 +2,71 @@
 
 'use strict';
 
+const commandLineArgs = require('command-line-args');
+const commandLineUsage = require('command-line-usage');
 const readline = require('readline');
 
 const Cli = require('../lib/cli/cli');
 
 let rl = null;
 
+const options = [
+  { name: 'verbose', alias: 'v', type: Boolean, multiple: true },
+  { name: 'no-reconnect', alias: 'R', type: Boolean, default: false },
+  { name: 'heartbeat-interval', alias: 'i', type: Number, default: 0 },
+  { name: 'help', alias: 'h', type: Boolean, default: false },
+];
+
+const usage = [
+  {
+    header: 'JSTP CLI client',
+  },
+  {
+    header: 'Options',
+    optionList: [
```

Comment:
You can use the `options` array here if you move `description` and `typeLabel` fields there.